pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -28,4 +28,4 @@ buildGoModule rec {
     license = licenses.asl20;
     maintainers = with maintainers; [ rlupton20 edude03 saschagrunert Frostman ];
   };
-}
+}
```

Comment:
It'd be nice if you configured your editor to leave these alone, as they're generally standard in most codebases, including nix.

Reply:
Ironically enough - This was automated :D.